OTVision/helpers/files.py

```diff
@@ -268,7 +270,9 @@ def denormalize_bbox(
 
 # TODO: Type hint nested dict during refactoring
 def normalize_bbox(
-    otdict: dict, keys_width: list[str] = None, keys_height: list[str] = None
+    otdict: dict,
+    keys_width: Union[list[str], None] = None,
+    keys_height: Union[list[str], None] = None,
 ):
```

Comment:
I think a type hint for the return is missing

Reply:
Ok reflected the request in the latest commit! Thanks!